azavea/raster-foundry
# Remove color corrections from datasources

## Overview
Remove default color corrections from datasources
Set default bands to datasource `natural` mode, or (1,2,3) by default

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Notes

We should consider removing the `value` property from the datasource composite json structure, since it can cause issues in `Circe` as seen in this PR

## Testing Instructions
* Run migrations
 * Add a scene to a new project
* Verify that its color correction is initialized with the datasource natural bands selected
* Verify that if the datasource does not have a natural band set, it uses 1,2, and 3 as rgb bands by default
* Verify that the color corrections have been removed from datasources in the UI and database

Closes #1788 
Closes #1643
